core/domain/exp_services.py

```diff
@@ -1249,6 +1622,11 @@ def clear_search_index():
 
 
 def index_explorations_given_ids(exp_ids):
+    """Indexs the exloration corresponding to the given exploration ids.
```

Comment:
Indexes

exploration --> explorations

(**Lots** of misspellings! Please proofread your work.)